docs/guides/sortable-image-tags.md

```diff
@@ -65,8 +66,8 @@ The branch and tag will usually be made available in a CI platform as environmen
 
 ## Example of a build process with timestamp tagging
```

Comment:
This is an awesome usage of `run_number` with GitHub actions, but perhaps we would want to add this as a separate example?

The header here already states that the example is specifically showing timestamps as a monotonically increasing number.

Reply:
Thanks, I made it into a separate example!